src/uct/base/uct_mem.c

```diff
@@ -137,22 +138,26 @@ ucs_status_t uct_mem_alloc(void *addr, size_t min_length, unsigned flags,
 
         case UCT_ALLOC_METHOD_THP:
 #ifdef MADV_HUGEPAGE
+            /* Fixed option is not supported for thp allocation*/
```

Comment:
minor: s/thp/the

Reply:
here, thp is acronym for "transparent huge pages"